tests/main/views/test_audits.py

```diff
@@ -148,7 +148,7 @@ def test_get_audit_event_for_missing_object_returns_404(self):
         self.add_audit_events_with_db_object()
 
         response = self.client.get('/audit-events?object-type=suppliers&object-id=100000')
-        data = json.loads(response.get_data())
+        json.loads(response.get_data())
```

Comment:
We probably don't need this line at all if the test never checks what's in the response JSON - you've just deleted the line elsewhere where this happens.